uint32 id
time start_time
float64 duration
geometry_msgs/Point center

# TODO: more metadata?
GazeDataPoint[] raw_data
